src/createGetHostedLibFileUrl.js

```diff
@@ -16,8 +16,17 @@
  * @param {string} hostedLibFilesBaseUrl
  * @returns {Function}
  */
+var state = require('./state');
+
 module.exports = function(hostedLibFilesBaseUrl) {
+  var buildInfo = state.getBuildInfo();
+
   return function(file) {
-    return hostedLibFilesBaseUrl + file;
+    var fileParts = file.split('.');
+    if (buildInfo['minified']) {
+      fileParts.splice(1, 0, 'min');
```

Comment:
You need to insert `min` before the last file part instead of after the first (e.g., a.b.js should turn into a.b.min.js and not a.min.b.js).

Also, even though we are currently requiring extension devs to end the file with `.js`, you might consider making it robust enough to have a filename without any periods, in which case you need to see what filename the server would produce under that circumstance and reproduce its behavior. For example, if the file name were `a`, does it produce `a.min`?

Also, you can avoid unnecessary processing if you structure your code like:

```
  return function(file) {
    if (buildInfo.minified) {
      var fileParts = file.split('.');
      fileParts.splice(1, 0, 'min');
      file = fileParts.join('.')
    }

    return hostedLibFilesBaseUrl + file;
  };
```